docs/setup.md

```diff
@@ -0,0 +1,171 @@
+[![](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/de.fayard/refreshVersions/de.fayard.refreshVersions.gradle.plugin/maven-metadata.xml.svg?label=refreshVersions)](https://plugins.gradle.org/plugin/de.fayard.refreshVersions)
+![Gradle Version](https://img.shields.io/endpoint?url=https%3A%2F%2Fgradle-latest-version-8xb9v8uk09jm.runkit.sh%2F)
```

Comment:
Some magic with shields at #242

Reply:
Yes!, look at the newest common, there is yet another markdown extension that allows us to use variables in markdown

```
// MARKDOWN
`$ ./gradlew wrapper --gradle-version {{version.gradle}}`


// YAML file
extra:
  version:
    gradle: 6.6.1
    refreshVersions: '0.9.6'
    refreshVersionsDev: '0.9.6-dev-007'

```